content/en/sidecar-container.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. 
+An application deployed on a second container will run side by side and share its lifecycle with a primary application running on a main container.
+
+## Problem it addresses
+
+There are scenarios where it is convenient to group [containers](/container/) to deal with problems related to the need to support cross-platform requirements or shared functionality. 
```

Comment:
```suggestion
There are scenarios where it is convenient to group [containers](/container/) to deal with cross-platform requirements or shared functionality. 
```

Reply:
@iamNoah1 accepted your review and recommendations. Added a couple of examples for the problems. All in commit https://github.com/cncf/glossary/commit/5e5ef2e9878569397127c4a065430d562000dce9 .